server/api/controllers/baptisms/update-baptism.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
+  friendlyName: "Update baptisms",
+
+  description: "Update baptisms",
+
+  inputs: {
+    params: {
+      required: false,
+      type: "json",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "Baptism records updated successfully",
+    },
+    invalid: {
+      description: "Failed to update baptism record",
+    },
+
+    missingRequiredFields: {
+      statusCode: 409,
+      description: "Please fill in the required fields.",
+    },
+
+    invalidBaptismId: {
+      statusCode: 409,
+      description: "The baptismId is invalid",
+    },
+  },
+
+  fn: async function ({ params }, exits) {
+    const { id: baptismId, ...toUpdate } = params;
+    if (baptismId) {
+      try {
+        let data = await Baptism.updateOne({
+          _id: baptismId,
+          isDeleted: false,
+        }).set(toUpdate);
+        if (data != null) {
+          return exits.success(data);
+        }
+        return exits.invalid();
+      } catch (err) {
+        sails.log.error(err);
+        return exits.error(err);
+      }
+    }
+    sails.log.error("missingRequiredFields");
```

Comment:
Mayb add a `sails.log.error(err)` too so that if it does happen, can easily debug

Reply:
For the other update APIs too